modules/centrality-algorithms/index.adoc

```diff
@@ -0,0 +1,22 @@
+= Centrality Algorithms
+:description: Overview of centrality algorithms.
+
+Centrality algorithms are one way to determine the importance of a vertex to a graph, based on the importance of a vertex to other vertices.
```

Comment:
Centrality algorithms find the important vertices in a graph based on their connections with other vertices.